tests/manage/noobaa/test_create_bucket.py

```diff
@@ -53,7 +55,7 @@ class TestBucketCreation:
         condition=config.ENV_DATA['platform'] != 'AWS',
         reason="Tests are not running on AWS deployed cluster"
     )
-    def test_s3_bucket_creation(self, noobaa_obj, buckets):
+    def s3_bucket_creation(self, noobaa_obj, buckets):
```

Comment:
Please use this:  @pytest.mark.skip(reason="")
http://doc.pytest.org/en/latest/skipping.html

Reply:
done